app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -31,69 +29,81 @@
         <div class='my2' id='front_target'></div>
       </div>
 
       <%# ---- Back Image ----- %>
 
-      <hr class="mt3 mb3"/>
-      <div class="back-image">
-        <h2>
-          <%= t('doc_auth.headings.upload_back_html') %>
-        </h2>
-
-        <%= accordion('totp-info-back', t('doc_auth.tips.title_html')) do %>
-          <%= render 'idv/doc_auth/tips_and_sample' %>
-          <div class='center'>
-            <%= image_tag(asset_url('state-id-sample-back.jpg'), height: 338, width: 450) %>
-          </div>
-        <% end %>
+      <div class="back-image mt3">
+        <p>
+          <span class='bold'>
+            <%= t('doc_auth.headings.document_capture_back') %><br/>
+          </span>
+          <%= t('doc_auth.tips.document_capture_hint') %>
+        </p>
 
         <%= f.input :back_image_data_url, as: :hidden %>
-        <%= f.input :back_image, label: false, as: :file, required: true, wrapper_class: 'mt3 sm-col-8' %>
+        <%= f.input :back_image, label: false, as: :file, required: true %>
         <div class='my2' id='back_target'></div>
       </div>
 
       <%# ---- Selfie ----- %>
       <% if liveness_checking_enabled? %>
         <hr class="mt3 mb3"/>
         <div class="selfie">
-          <h2>
-            <%= t('doc_auth.headings.selfie') %>
-          </h2>
+          <p><%= t('doc_auth.instructions.document_capture_selfie_instructions') %></p>
+
+          <p class="mt2 mb0"><%= t('doc_auth.tips.document_capture_header_text') %></p>
+          <ul>
+            <li><%= t('doc_auth.tips.document_capture_selfie_text1') %></li>
+            <li><%= t('doc_auth.tips.document_capture_selfie_text2') %></li>
+            <li><%= t('doc_auth.tips.document_capture_selfie_text3') %></li>
+          </ul>
+
+          <p class='mt3'>
+            <span class='bold'>
+              <%= t('doc_auth.headings.document_capture_selfie') %><br/>
+            </span>
+            <%= t('doc_auth.tips.document_capture_hint') %>
+          </p>
 
           <%= f.input :selfie_image_data_url, as: :hidden %>
-          <%= f.input :selfie_image, label: false, as: :file, required: true, wrapper_class: 'mt3 sm-col-8' %>
+          <%= f.input :selfie_image, label: false, as: :file, required: true %>
           <div class='my2' id='selfie_target'></div>
         </div>
       <% end %>
 
       <%# ---- Submit ----- %>
 
-      <div class='mt3'>
+      <div class='mt4 mb4'>
         <%= render 'idv/doc_auth/submit_with_spinner' %>
       </div>
-    <% end %>
+    <% end %> <%# end simple_form_for %>
 
-    <p class='mt3 mb0'><%= t('doc_auth.info.upload_image') %></p>
+    <p class='mt3 mb0'><%= t('doc_auth.info.document_capture_upload_image') %></p>
 
     <%= javascript_pack_tag 'image-preview' %>
   </div>
   <%= render 'idv/doc_auth/start_over_or_cancel' %>
-  <%= javascript_pack_tag 'document-capture' %>
+  <% unless Figaro.env.document_capture_react_enabled == 'false' %>
```

Comment:
Should we have defaults for this in `application.yml.default`, even if it's largely something you'd intend to be controlled in very local-dev-environment circumstances, or be temporary?

(The answer may well be "No". I'm just not sure if there's conventions to have the `.default` file exhaustively list all of the variables in use)